types/domain.ts

```diff
@@ -0,0 +1,9 @@
+import { DbtExposure } from "./dbt/exposure";
+
+export interface ExposureMetaData extends DbtExposure {
+  path: string;
+  unique_id: string;
```

Comment:
This is also present in DbtExposure?

What's the logic behind splitting this, as I think some of the data here is coming from dbt as well?

Reply:
These are extra attributes added on top of dbt type from manifest